src/ol_infrastructure/applications/airbyte/__main__.py

```diff
@@ -186,14 +186,13 @@
 rds_defaults["instance_size"] = (
     airbyte_config.get("db_instance_size") or rds_defaults["instance_size"]
 )
-rds_password = read_yaml_secrets(
-    Path(f"data/airbyte/rds.{stack_info.env_suffix}.yaml")
-)["password"]
+
+rds_password = airbyte_config.require("rds_password")
 
 airbyte_db_config = OLPostgresDBConfig(
     instance_name=f"airbyte-db-{stack_info.env_suffix}",
     password=rds_password,
-    storage=airbyte_config.get("db_capacity"),
+    storage=airbyte_config.get("db_capacity") or "334",
```

Comment:
Probably worth moving that `334` into the magic numbers module with either a descriptive name or a comment to describe where that number comes from. Probably also worth just adding that into the defaults dictionary.